bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/TeaserImpl.java

```diff
@@ -126,6 +128,11 @@ private void initModel() {
             targetPage = pageManager.getPage(linkURL);
         }
 
+        if (pretitleHidden) {
+            pretitle = null;
```

Comment:
I would not change the value of the `pretitle` if it is hidden: the value should be the one defined by the author, even if the field is hidden.

Reply:
I agree, but we already do the same for the title and description and `pretitleHidden` isn't exposed to the template so the only way to check for display is `? teaser.pretitle`. Perhaps it's best to be consistent here with the current behaviour?

Reply:
hm... I agree that we should be consistent and have the same behavior as for `title` and `description`.

Reply:
Yes, I tried to be consistent and have the same behavior as for the other fields.